documentation/source/user_section/installation/windows.rst

```diff
@@ -67,11 +63,9 @@ The easiest way to try out different versions of SCT is using Git.
 2. Installing SCT
 *****************
 
-We recommend that you install SCT into a Python virtual environment. To help with this process, SCT provides an installer script that will automate the process of creating a Python virtual environment for you.
-
-1. Navigate to the `Releases page <https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/>`_ , then download the ``install_sct.bat`` script from the "Assets" section of the latest release. 
+1. Navigate to the `Releases page <https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/>`_ , then download the ``install_sct-<version>_win.bat`` script from the "Assets" section of the latest release.
 
-2. Run the script by double-clicking it. The script will fetch the SCT source code, then install the `spinalcordtoolbox` package into a Python virtual environment for you.
+2. Run the script by double-clicking it. The script will fetch the SCT source code, then install the `spinalcordtoolbox` package into a Miniconda environment for you.
 
 3. Once the installer finishes, follow the instructions given at the end of the Command Prompt window, which will instruct you to add SCT to your PATH.
```

Comment:
TODO: Update these instructions (and the `install_sct.bat` script logging) to mention how to set `$SCT_DIR` as well, as per https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4146#issuecomment-1624226742.